tests/core/test_settings.py

```diff
@@ -278,25 +278,25 @@ def test_encoding_absent_in_v2(is_v1, mock_conf):
 )
 def test_get_traceback_session_if_aip_44_enabled():
     # ensure we take the database_access_isolation config
-    InternalApiConfig._init_values()
-    assert InternalApiConfig.get_use_internal_api() is True
+    try:
+        InternalApiConfig._init_values()
+        assert InternalApiConfig.get_use_internal_api() is True
 
-    # ensure that the Session object is TracebackSession
-    configure_orm()
+        with create_session() as session:
+            assert isinstance(session, TracebackSession)
 
-    from airflow.settings import Session
+            # no error just to create the session
+            # but below, when we try to use, it will raise
 
-    assert Session == TracebackSession
```

Comment:
removed this part of the test so we can avoid reconfigure orm

it will me an that the behavior that `settings.Session` is traceback session when using db isolation will be _untested_ -- but, maybe that's acceptable